cores/esp8266/FSImpl.h

```diff
@@ -51,6 +51,8 @@ class FileImpl {
     // as the FS is allowed to return either the time of the last write() operation or the
     // time present in the filesystem metadata (often the last time the file was closed)
     virtual time_t getLastWrite() { return 0; } // Default is to not support timestamps
+    // Same for creation time.
+    virtual time_t getCreation() { return 0; } // Default is to not support timestamps
```

Comment:
The function name seems a bit confusing to me. Consider getCreationTime() to be explicit.

Reply:
Maybe other getters of time values should also get a Time posfix to be clear.

Reply:
> ... getters of time values should also get a Time posfix to be clear.

This name is actually an ESP32-ism, part of their API I tried to copy when consolidating the Filesystems.  I suppose we could deprecate this one and direct to a `getLastWriteTime`, but then we'll differ.  I like the Time postfix myself...

Reply:
Updated in latest pushed.